docs/pages/docs/features/caching.mdx

```diff
@@ -75,6 +75,48 @@ You can also always disable caching on a specific task by setting `cache` to `fa
 }
 ```
 
+## Alter Caching Based on Environment Variables
+
+You can control caching based on the value of environment variables by setting `env` key where applicable.
+
+**At the root** will tell `turbo` to include the hashed values of each environment variable it all other hashes.
+**In the `pipeline`** will tell `turbo` to inclued the hashed values of each environment variable on a per-task or per-package-task basis.
+
+```json
+{
+  "turbo": {
+    "baseBranch": "origin/master",
+    "env": [ // values will impact the hashes of all tasks
+      "GITHUB_TOKEN"
+    ],
+    "pipeline": {
+      "build": {
+        "dependsOn": {
+          "^build"
+        },
+        "outputs": ["dist/**"]
+        "env": [ // values will impact hashes of all "build" tasks
```

Comment:
Does this combine the values of `turbo.env` with the ones defined in the task or are they getting override? Would this end up with `["GITHUB_TOKEN", "SOME_ENV_VAR"]` or `["SOME_ENV_VAR"]`?